resources/variants/kingroon_base_0.6.inst.cfg

```diff
@@ -0,0 +1,12 @@
+[general]
```

Comment:
The base printer is never meant to be visible, so it doesn't need variants. There are variants for the Kingroon KP3 and KP3S specifically.